src/sentry/static/sentry/app/views/projectDebugFiles.jsx

```diff
@@ -54,11 +58,51 @@ const DebugSymbolDetails = styled.div`
   margin-top: 4px;
 `;
 
+const formFields = [
+  {
+    name: 'builtinSymbolSources',
+    type: 'select',
+    multiple: true,
+    label: t('Built-in Repositories'),
+    help: t(
+      'Configures which built-in repositories Sentry should use to resolve debug files.'
+    ),
+    choices: [['microsoft', t('Microsoft Symbol Server')]],
+  },
+  {
+    name: 'symbolSources',
+    type: 'string',
+    label: t('Custom Repositories'),
+    placeholder: t('Paste JSON here.'),
+    multiline: true,
+    autosize: true,
+    maxRows: 10,
```

Comment:
can we make this the same as the grouping configs: `inline: false`, `monospace: true` and not save on blur?

For reference: https://github.com/getsentry/sentry/blob/0db91ef4049866126b0e33817c3e3c65f6e27f97/src/sentry/static/sentry/app/data/forms/projectGeneralSettings.jsx#L179-L187

Reply:
Updated, see new screenshot. However, as stated in the description this is only temporary UI ;)